README.adoc

```diff
@@ -10,6 +10,14 @@
 :KRAMDOWN_ASCIIDOC_VERSION: 2.0.0
 :ASCIIDOCTOR_BIBTEX_VERSION: 0.8.0
 :ASCIIDOCTOR_KROKI_VERSION: 0.5.0
+:ASCIIDOCTOR_WEB_PDF_VERSION: v1.0.0-alpha.14
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
```

Comment:
What is the reason of this addon?

Reply:
[Demo of some useful tips for using Asciidoc on GitHub](https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74#admonitions) -- show the admonitions as icons.

Reply:
Oh this is cool! 

Do you mind adding a description to the PR mentioning it with a screenshot (for later readers)?

Otherwise great improvement for this

Reply:
OK. I will add the description later. 

*I must sleep, and have to work tomorrow.*

Reply:
Many thanks for your time ! Take the time needed for this contribution.